src/common/routing/RouteDefs.jsx

```diff
@@ -40,12 +40,11 @@ const RouteDefs = () => {
         <Routes>
           <Route path="/" element={<Home />} />
           <Route path="/tech-stacks" element={<TechStack />} />
+          <Route path="/editplay/:username/:playname" element= {<CreatePlay />} />
           <Route path="/plays" element={<App />}>
             <Route index element={<PlayList />} />
-            <Route exact path="create" element={<CreatePlay />} />
-            {process.env.NODE_ENV === "development" && (
-              <Route exact path="created/:playid" element={<PlayCreated />} />
-            )}
+            <Route exact path="create" element= {<CreatePlay />}/>
+            {process.env.NODE_ENV === "development" && <Route exact path="created/:playid" element={<PlayCreated />} />}
```

Comment:
Not related to current PR however, i feel the paths should be
- /play/create
- /play/create/success
- /play/edit/slug

Reply:
yup good to handle it other PR